lib/router/router.js

```diff
@@ -97,11 +97,11 @@ export default class Router {
     window.history.back()
   }
 
-  push (route, url) {
+  push (route, url = route) {
```

Comment:
Set the default value for url. So, we can access it like `Router.push()` if we don't change paths with #310